README.md

```diff
@@ -134,33 +185,51 @@ In `docker-compose.yml`, the following ports on the docker host are connected t
 - Node-red runs on 1880/tcp.
 
 - Postfix runs on 25/tcp.
+- Apiserver runs on the below ports
+
+  - DNC-server runs on 8891/tcp.
+
+  - DNC-Std-Plugin runs on 8892/tcp.
+
+  - DNC-GI-Plugin runs on 8893/tcp.
+
+- Expo runs on 19006/tcp.
+
+- Mongodb runs on 27017/tcp.
+
+Remember, if the server is running on a cloud platform like Microsoft Azure or AWS, one needs to check the firewall and confirm that the ports are open to the outside world.
 
 ## Data Files
 
 When designing this collection of services, there were two choices to store the
 data files:
 
-- we could keep them inside the docker containers, or
+- We could keep them inside the docker containers, or
```

Comment:
Normal US style is not to capitalize these bullets, as they're not the beginning of sentences.

Reply:
Ok, It's done.